docs/OntologiesReviewWorkflow

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc
+title: Ontology Review Process
+---
+
+This page deals with the process, policies, and guidelines for manually reviewing newly-submitted ontologies, with particular focus on operational aspects such as:
+
+- which ontologies are reviewed, and in what order?
+- how are reviewers chosen?
+- what is the workflow for conducting a review?
+- what are the criteria used?
+
+Ontologies are reviewed manually after completing the automated evaluation. Though the automated evaluation checks for adherence to [OBO Foundry principles](http://www.obofoundry.org/principles/fp-000-summary.html), it cannot capture certain nuances outlined in the principles nor evaluate ontology quality. The purpose of the manual review is to check the ontology for issues that the NOR Dashboard does not cover.
+
+# Steps prior to manual review
+
+Once an ontology is submitted to the OBO Foundry, it is added to the New Ontology Dashboard for automated evaluation. The steps involved in ontology submission and subsequent dashboard evaluation are described in detail in this [FAQ answer](http://obofoundry.org/faq/how-do-i-register-my-ontology.html) and links therein.
+
+# Review priority
+
+All submitted ontologies will be manually reviewed based on the order in which the ontology becomes compliant with the automated evaluation (that is, when the [New Ontology Request (NOR) Dashboard](http://obofoundry.org/obo-nor.github.io/dashboard/index.html) status is set to 'pass').
```

Comment:
"is set to" suggests misleadingly that someone sets it. Maybe say "...status is 'pass'"?

Reply:
I don't see it the same way (as a program can change statuses too) but I changed it anyway.